github/GithubObject.py

```diff
@@ -63,15 +63,16 @@
     from .Requester import Requester
 
 T = typing.TypeVar("T")
+T_co = typing.TypeVar("T_co", covariant=True)
 
 
-class Attribute(Generic[T]):
+class Attribute(Protocol[T_co]):
```

Comment:
this make `Attribute[str | None]` and `Attribute[str] | Attribute[None]` to be the same type